src/theme/foundations/colours.ts

```diff
@@ -104,6 +105,20 @@ export const colours: { [k in IsomerColorScheme]: NestedRecord } = {
       disabled: "#BFBFBF",
     },
   },
+  brand: {
+    illustration: {
+      50: "#FBF8FF",
+      100: "#EFE7FF",
+      200: "#C9B3FF",
+      300: "#AA90FA",
+      400: "#8C73DB",
+      500: "#6D58BB",
+      600: "#5543A4",
+      700: "#493897",
+      800: "#3B2E8A",
+      900: "#271E78",
+    },
+  },
```

Comment:
had some comments for @NatMaeTan - will leave it to her to follow up + jot notes TIA nat